libfaad/sbr_hfgen.c

```diff
@@ -295,16 +295,23 @@ static void auto_correlation(sbr_info *sbr, acorr_coef *ac, qmf_t buffer[MAX_NTS
 
     exp = wl_min_lzc(mask);
 
+    /* All-zero input. */
+    if (exp == 0) {
+        RE(ac->r12) = 0;
+        IM(ac->r12) = 0;
+        RE(ac->r22) = 0;
```

Comment:
How about `IM(ac->r22)`?

Reply:
`IM` part of `r22` is never accessed,... Same with `IM(r11)`. Likely those are pure-real due to the nature of transform.

But thanks for pointing raising attention. Actually there as more assignments after `if-def` section!